src/Mobs/Monster.cpp

```diff
@@ -1294,3 +1401,60 @@ cMonster::eFamily cMonster::GetMobFamily(void) const
 {
 	return FamilyFromType(m_MobType);
 }
+
+
+
+
+
+void cMonster::LeashTo(cEntity * a_Entity, bool a_ShouldBroadcast)
+{
+	m_LeashedTo = a_Entity;
```

Comment:
You mean check if already leashed and if so then do nothing, (and viceversa) right?